wiki/plugins/notifications/models.py

```diff
@@ -27,7 +27,7 @@ class Meta:
 def default_url(article, urlpath=None):
     try:
         if not urlpath:
-            urlpath = wiki_models.URLPath.objects.get(articles=article)
+            urlpath = wiki_models.URLPath.objects.get(article=article)
```

Comment:
`articles` is a ReverseGeneric field on the model to `ArticleForObject` objects which point to the `URLPath`. However, we are passing an `Article` object against the parameter. I suspect this worked because for every article there exists a corresponding `ArticleForObject` object and so the ids matched. However, in Django 1.8 `get` throws an error that it is expecting an `Article` object. So I have changed this to match against the `article` field which contains the most recent `Article` object which points to this `URLPath` object.

https://github.com/edx/django-wiki/blob/django1.8-upgrade/wiki/models/urlpath.py#L230-L237